ocp_resources/network_attachment_definition.py

```diff
@@ -197,3 +197,42 @@ def to_dict(self):
     @property
     def resource_name(self):
         return f"ovs-cni.network.kubevirt.io/{self.bridge_name}"
+
+
+class FlatL2NetworkAttachmentDefinition(NetworkAttachmentDefinition):
+    def __init__(
+        self,
+        network_name=None,
+        teardown=True,
+        *args,
+        **kwargs,
+    ):
+        """
+        Create and manage a flat-l2 NetworkAttachmentDefinition.
+        API reference:
+        https://docs.openshift.com/container-platform/4.13/networking/multiple_networks/configuring-additional-network.html#configuration-ovnk-network-plugin-json-object_configuring-additional-network
+
+        Args:
+            network_name (str, default: None): The name of the network. This attribute is not namespaced.
+            teardown (bool, default: True): Indicates if this resource would need to be deleted.
+        """
+        super().__init__(
+            cni_type="ovn-k8s-cni-overlay",
+            teardown=teardown,
+            *args,
+            **kwargs,
+        )
+        self.topology = "layer2"
+        self.netAttachDefName = f"{self.namespace}/{self.name}"
+        self.network_name = (
+            f"{network_name}-network" if network_name is None else network_name
+        )
+
```

Comment:
```
In [1]: network_name=None

In [2]: (
   ...:             f"{network_name}-network" if network_name is None else network_name
   ...:         )
Out[2]: 'None-network'
```
this name does not make sense

Reply:
Currect, thanks! I meant to use "name" and not "network_name" in the beginning. It doesn't make sense here now that we don't pass the name.
I'll refactor and move it to the to_dict function.